Makefile

```diff
@@ -87,6 +87,26 @@ initdirs:
 	mkdir -p httpd
 	mkdir -p mig
 	mkdir -p state
+	mkdir -p volatile/mig_system_run
+	mkdir -p volatile/openid_store
+	mkdir -p persistent/freeze_home
+	mkdir -p persistent/mrsl_files
+	mkdir -p persistent/resource_home
```

Comment:
A bunch of the created persistent sub directories are not used as part of any volume, is this just preparation for future volumes?

Reply:
Good catch!
No, the point was to pull out _all_ the `state` subdirs that get populated with actual site data into `persistent` and use them from there, so that one can safely wipe `state` without losing data. 
I'll add the corresponding missing entries for `freeze_home` , `mrsl_files` , `resource_home` , `sharelink_home` and `sss_home` when replacing the hard-coded `log` and `persistent` dir names with `LOG` and `PERSISTENT` variables as we discussed off-list.